src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
Remove this constructor or add field initialization. Your Ball class can produce unexpected behavior by creating a ball with null color or/and balls with a similar ballNumber. The default value for String is null. The default value for primitives is 0 (or 0.0 for floating point values).